src/test/java/ca/team2706/frc/robot/commands/drivebase/CurvatureDriveTest.java

```diff
@@ -0,0 +1,115 @@
+package ca.team2706.frc.robot.commands.drivebase;
+
+import ca.team2706.frc.robot.subsystems.DriveBase;
+import com.ctre.phoenix.CTREJNIWrapper;
+import com.ctre.phoenix.motorcontrol.SensorCollection;
+import com.ctre.phoenix.motorcontrol.can.MotControllerJNI;
+import com.ctre.phoenix.motorcontrol.can.WPI_TalonSRX;
+import com.ctre.phoenix.sensors.PigeonIMU;
+import edu.wpi.first.wpilibj.AnalogInput;
+import edu.wpi.first.wpilibj.Notifier;
+import edu.wpi.first.wpilibj.PWM;
+import edu.wpi.first.wpilibj.drive.DifferentialDrive;
+import mockit.Expectations;
+import mockit.Injectable;
+import mockit.Mocked;
+import mockit.Verifications;
+import org.junit.Before;
+import org.junit.Test;
+import util.Util;
+
+import java.util.function.Supplier;
+
+public class CurvatureDriveTest {
+    @Mocked
+    private Supplier<Double> forwardVal;
```

Comment:
This isn't going to work the way you probably want it to. Since you're mocking a `Supplier<Double>` below as well, it won't care which instance the methods are called on.

I would advise using `@Injectable` here instead since that's really what it's meant for.